src/imageLoader/createImage.js

```diff
@@ -173,12 +182,24 @@ function createImage(imageId, pixelData, transferSyntax, options = {}) {
           canvas.width = imageFrame.columns;
 
           const context = canvas.getContext('2d');
-          const imageData = context.createImageData(
-            imageFrame.columns,
-            imageFrame.rows
-          );
 
-          convertColorSpace(imageFrame, imageData);
+          let imageData;
+
+          if (useRGBA) {
+            imageData = context.createImageData(
+              imageFrame.columns,
+              imageFrame.rows
+            );
+          } else {
+            const buffer = new ArrayBuffer(
+              3 * imageFrame.columns * imageFrame.rows
+            );
+
+            imageData = new Uint8ClampedArray(buffer);
+          }
+
+          convertColorSpace(imageFrame, imageData, useRGBA);
+
           imageFrame.imageData = imageData;
           imageFrame.pixelData = imageData.data;
```

Comment:
Isn't imageData.data now invalid if imageData is a Uint8ClampedArray?

There's also no reason to keep the imageData property if we aren't using rgba